ci_test/unit_tests/test_unit_layer_convolution_distconv.py

```diff
@@ -292,5 +293,7 @@ def construct_data_reader(lbann):
 # Note: Create test name by removing ".py" from file name
 _test_name = os.path.splitext(os.path.basename(current_file))[0]
 for _test_func in tools.create_tests(setup_experiment, _test_name,
-                               environment=tools.get_distconv_environment()):
+                                     environment=lbann.contrib.args.get_distconv_environment(),
+                                     skip_clusters=["tioga"],
```

Comment:
This is already tracked in #2326. It might be good to add more detail to that ticket regarding the manner of failure.